src/core/core.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { HeaderComponent } from './header/header.component';
+import { FooterComponent } from './footer/footer.component';
+import { Routes, RouterModule } from '@angular/router';
+
+
+@NgModule({
+    imports: [],
```

Comment:
follow the same order in the module.ts.
declaration, imports and exports or this one.